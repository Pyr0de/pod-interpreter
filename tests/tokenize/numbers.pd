12
12.5
15.13252123
0.231
754.34
0.523.93
12365434
