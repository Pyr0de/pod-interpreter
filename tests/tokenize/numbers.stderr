ExitCode: 0
[line 6] Improper Float: 0.523.93
